src/cljs/webchange/ui_framework/components/icon/index.cljs

```diff
@@ -74,6 +80,7 @@
    "expand-arrow-down"  expand-arrow-down/data
    "expand-arrow-right" expand-arrow-right/data
    "expand"             expand/data
+   "eye"                eye/data
```

Comment:
we already have icons:
eye -> visibility-on | visibility-off
plus -> add
trash -> remove